packages/odyssey/src/scss/base/_iconography.scss

```diff
@@ -0,0 +1,13 @@
+.ods-icon {
+  position: relative;
+  top: -0.1em;
+  width: 1em;
+  height: 1em;
+  vertical-align: middle;
+}
+
+.ods-button {
```

Comment:
This also seems weird to me. Developers will need to import `iconography` in order to use this within a button. Why can't this live within `components/button`?

Reply:
I think it will eventually. For now, I'm trying to stick to the conventions in place rather than continue to maintain things like `_buttons-context` and `_buttons-layout` alongside.

@arnoldsandoval-okta - I'm thinking we should get an issue logged to scope out the refactoring here (not much work) and standardize on a solution (much work). Then we can run the pitch past @jmelberg-okta, @jeffbaltazar-okta, and friends to ensure both teams needs are accounted for. Sound good for all?